pkg/config/config.go

```diff
@@ -530,6 +530,15 @@ func GetMultipleEndpoints() (map[string][]string, error) {
 	return getMultipleEndpointsWithConfig(Datadog)
 }
 
+// GetBatcherLimit returns the mximum amount of elements per batcher for the batcher
+func GetBatcherLimit() int {
```

Comment:
rename accordingly to name chosen in TopologyBuilder

Reply:
what would you call the config item?